packages/form-js-viewer/src/render/components/form-fields/Text.js

```diff
@@ -11,6 +11,10 @@ const type = 'text';
 
 
 export default function Text(props) {
+
+  const form = useService('form');
+  const { useTargetBlank } = form._getState().properties;
```

Comment:
I think at least one thing we should make sure is that we properly pipe viewer (and editor) `properties` via the playground: https://github.com/bpmn-io/form-js/blob/develop/packages/form-js-playground/src/components/PlaygroundRoot.js#L95

As a follow-up, we should also create issues in `form-playground` and `Tasklist` to either set the flag or, in the case of the form-playground, piping through `properties` as well.

Reply:
Just checked the form-playground, seems like we are good as we already pipe all the props down there: https://github.com/camunda/form-playground/blob/main/src/index.js#L139.

Might make sense to at least add some docs over there. I added an issue for this: https://github.com/camunda/form-playground/issues/67